index.html

```diff
@@ -146,58 +168,71 @@ <h1 class="about">About me</h1>
-                </div>
-                <ul class="main-ul">
-                    <li>Javascript</li>
-                    <li>Ruby</li>
-                    <li>Html</li>
-                    <li>Css</li>
-                </ul>
-            </li>
-            <li class="headli">
-                <div><img src="Images/Rectangle56.png" class="rectangle56" alt="shape">
-                    <p class="alignim">Frameworks</p>
-                </div>
-                <ul class="main-ul">
-                    <li>Bootstrap</li>
-                    <li>Ruby on Rails</li>
-                    <li>RSPec</li>
-                    <li>CapyBara</li>
-                    <li>Selenium</li>
-                </ul>
-            </li>
-            <li class="headli">
-                <div><img src="Images/Rectangle57.png" class="rectangle57" alt="shape">
-                    <p class="alignim">Skills</p>
-                </div>
-                <ul class="main-ul">
-                    <li>Codekit</li>
-                    <li>Github</li>
-                    <li>Codepen</li>
-                    <li>Gitlab</li>
-                    <li>Terminal</li>
-                </ul>
-            </li>
-        </ul>
+        <div class="langframe">
+            <ul class="main-ul lang">
+                <li class="headli">
+                    <div><img src="Images/Rectangle55.png" class="rectangle55" alt="rectangle">
+                        <p class="alignim">Language</p>
+                    </div>
+                    <ul class="main-ul lang">
+                        <li>Javascript</li>
+                        <li>Ruby</li>
+                        <li>Html</li>
+                        <li>Css</li>
+                    </ul>
+                </li>
+            </ul>
+            <ul class="main-ul">        
+                <li class="headli">
+                    <div><img src="Images/Rectangle56.png" class="rectangle56" alt="shape">
+                        <p class="alignim">Frameworks</p>
+                    </div>
+                    <ul class="main-ul">
+                        <li>Bootstrap</li>
+                        <li>Ruby on Rails</li>
+                        <li>RSPec</li>
+                        <li>CapyBara</li>
+                        <li>Selenium</li>
+                    </ul>
+                </li>
+            </ul> 
+            <ul class="main-ul skills">   
+                <li class="headli">
+                    <div><img src="Images/Rectangle57.png" class="rectangle57" alt="shape">
+                        <p class="alignim">Skills</p>
+                    </div>
+                    <ul class="main-ul skills2">
+                        <li>Codekit</li>
+                        <li>Github</li>
+                        <li>Codepen</li>
+                        <li>Gitlab</li>
+                        <li>Terminal</li>
+                    </ul>
+                </li>
+            </ul>
+        </div>
```

Comment:
Your design             |  Figma Design
:-------------------------:|:-------------------------:
![aboutme](https://user-images.githubusercontent.com/69638013/154555907-729bb03d-4a23-4de5-8579-5135fb0188d4.PNG) | ![good](https://user-images.githubusercontent.com/69638013/154555963-78cc9186-1661-4e9d-8096-0cb94e7f90b1.PNG)
- [ ] Kind resize about-me section background image to look like the original figma design
- [ ] Kindly align all   main-ul li tags to the left check the Figma design